build.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 name: "jellyfin-plugin-nextpvr"
 guid: "9574ac10-bf23-49bc-949f-924f23cfa48f"
-version: "1" # Please increment with each pull request
-jellyfin_version: "10.3.0" # The earliest binary-compatible version
+version: "2" # Please increment with each pull request
+jellyfin_version: "10.4-*" # The earliest binary-compatible version
```

Comment:
I'm not sure if this value supports wildcards, @joshuaboniface do you know?

Reply:
Hrm, I'm not 100% sure. I don't think it does, but there should be no need to wildcard this, since it's always a "this or later" conditional.